src/main/scala/temple/generate/server/go/service/GoServiceGenerator.scala

```diff
@@ -47,9 +47,24 @@ object GoServiceGenerator extends ServiceGenerator {
       File(s"${serviceRoot.name}/dao", "errors.go") -> GoServiceDAOGenerator.generateErrors(serviceRoot.name),
       File(s"${serviceRoot.name}/dao", "dao.go") -> mkCode.doubleLines(
         GoCommonGenerator.generatePackage("dao"),
-        GoServiceDAOGenerator.generateImports(serviceRoot.module),
-        GoServiceDAOGenerator.generateStructs(),
-        GoServiceDAOGenerator.generateInit(),
+        GoServiceDAOGenerator.generateImports(serviceRoot.attributes, serviceRoot.module),
+        GoServiceDAOGenerator
+          .generateDatastoreInterface(serviceRoot.name, serviceRoot.operations, serviceRoot.enumByCreatedBy),
```

Comment:
I’d prefer `GoServiceDAOGenerator.generateDatastoreInterface(\n` for consistency